pysmt/test/test_solving.py

```diff
@@ -240,6 +240,34 @@ def test_examples_btor(self):
             self.assertEqual(satisfiability, s, f)
 
     def do_model(self, solver_name):
+        def assert_model(f, subs):
```

Comment:
Would it make sense to have this code as a service of pysmt?

Something like `check_model(formula, dict_like_object)`?